src/navigator/SettingsMenu.tsx

```diff
@@ -130,39 +130,17 @@ export default function SettingsMenu({ route }: Props) {
 
   const sessionId = useSelector(sessionIdSelector)
   const devModeActive = useSelector(devModeSelector)
-  const numberVerified = useSelector(phoneNumberVerifiedSelector)
 
   useEffect(() => {
     if (AppAnalytics.getSessionId() !== sessionId) {
-      dispatch(setSessionId(sessionId))
+      dispatch(setSessionId(AppAnalytics.getSessionId()))
```

Comment:
i think session id wasn't used before, because this was the only place it was being set and it was always being set to empty 😅 i kept it here in case it is useful